rero_ils/modules/ill_requests/views.py

```diff
@@ -78,3 +90,57 @@ def get_patron(loc_pid):
 
     return render_template('rero_ils/ill_request_form.html',
                            form=form, viewcode=viewcode)
+
+
+def _populate_document_data_form(doc_pid, form):
+    """Populate document data in the form.
+
+    :param: doc_pid: Document PID.
+    :param: form: The ill form.
+    """
+    if not (doc := Document.get_record_by_pid(doc_pid)):
+        return
```

Comment:
For me, it will be more readable 
```python
doc = Document.get_record_by_pid(doc_pid)
if not doc:
    return
```
One line more, but much more comprehensible if you continue to use `doc` outside of the "if" statement